components/edit-collective/actions/Archive.js

```diff
@@ -150,7 +152,7 @@ const ArchiveCollective = ({ collective }) => {
       )}
       {!isArchived && collective.isHost && (
         <P color="rgb(224, 183, 0)" my={1}>
-          {isSelfHosted ? (
+          {isIndependentCollective ? (
```

Comment:
```suggestion
          {collective.type === CollectiveType.COLLECTIVE ? (
```

That would be simpler.

Reply:
You can remove the `isIndependentCollective` variable.

Reply:
This is done. 👍🏽